conf/ocsci/downstream_config.yaml

```diff
@@ -14,4 +14,11 @@ REPORTING:
   us_ds: 'DS'
   ocs_must_gather_image: "quay.io/rhceph-dev/ocs-must-gather"
 DEPLOYMENT:
-  ocs_operator_deployment: false
+  ocs_operator_deployment: true
+  ocs_operator_storage_cluster_cr: "http://pkgs.devel.redhat.com/cgit/containers/ocs-registry/plain/ocs_v1_storagecluster_cr.yaml?h=ocs-4.2-rhel-8"
+  ocs_csv_version_yaml: "http://pkgs.devel.redhat.com/cgit/containers/ocs-registry/plain/olm-catalog/ocs-operator/ocs-operator.package.yaml?h=ocs-4.2-rhel-8"
+  ocs_operator_olm: "http://pkgs.devel.redhat.com/cgit/containers/ocs-registry/plain/deploy-with-olm.yaml?h=ocs-4.2-rhel-8"
+  ocs_operator_cs_image: "quay.io/rhceph-dev/ocs-registry"
+  # Catalog source image tag can be overwritten by following parameter to the
+  # specific tag from ocs_operator_cs_image:
+  # ocs_operator_cs_tag: "4.2-32.9b6c93e.master"
```

Comment:
Do you mean to have whole `quay.io/rhceph-dev/ocs-registry:latest here` or `:specific-tag` ?

The intention was to have defined the image and the image tag separately and in case the image tag is not defined by the job we will use the latest one from call of this function: https://github.com/red-hat-storage/ocs-ci/pull/713/files#diff-153ccf5916ca230718b5455c349afbcdR145 .

This way we won't use the latest tag an the job/ocs-ci can later report which specific tag was used for execution as the latest is the floating tag.

We can have it in one parameter by default with `:latest` and check if the latest is in the string and then we will replace it with specific image tag. I thought that you will send us just the tag version in the UMB message, but if you will send us whole image we can do it like that.

Anyone this is the initial PR. The enhancement/alignment can be done right after as the things are not 100% clear yet.

Reply:
Yeah, this isn't a big deal as right now I'm doing [this](https://github.com/red-hat-storage/ocs-ci/pull/733/files#diff-f641b3a374762f7eb3504874b8d6aa28R100-R102), but it's something we can address in the future.

Reply:
done